website/www/site/data/en/community_list.yaml

```diff
@@ -0,0 +1,27 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+- title: { text: Join Beam Community, link: /community/join-beam }
+  body: Here you will find real life cases.
+  icon: icons/community/bee-icon.svg
+- title: { text: Contact us!, link: /community/contact-us }
+  body: You want to know what is going on? Raise a bug? Propose a feature request? Here you will find all of them info all of the info on who to contact.
+  icon: icons/community/envelope-icon.svg
+- title: { text: Keep up with Beam, link: / }
+  body: Check out our social media and be first to know about any updates.
+  icon: icons/community/message-icon.svg
+- title: { text: Meet up with us!, link: / }
+  body: Want to meet us in person? (or monitor-person). This is the place to go.
+  icon: icons/community/calendar-icon.svg
+- title: { text: Promotion, link: / }
+  body: Need some marketing materials? Here you will find it all.
+  icon: icons/community/box-icon.svg
```

Comment:
A couple of problems with this file:
- It looks like the copy in it wasn't updated in the last commit. Is this something you're still waiting on?
- The links for the last three items are `/` should they point somewhere else? Please add a TODO if we need to update these later.
- The link for the first item, /community/join-beam, doesn't exist. Maybe it's added in another PR? Or is this link just incorrect?

Reply:
Looks like join-beam is added in #13718